src/data/quizzes/questionBank.ts

```diff
@@ -1,1746 +1,165 @@
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 3 },
+  ],
+  wallets: [
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 2, correctAnswer: 2 },
+  ],
+  security: [
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 2, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 2, correctAnswer: 2 },
+  ],
+  nfts: [
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 2, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 2, correctAnswer: 2 },
+  ],
+  rollups: [
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 4 },
+  ],
+  merge: [
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 2, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 2 },
+  ],
+  daos: [
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 4 },
+  ],
+  "staking-solo": [
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 4 },
+  ],
+  scaling: [
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 2 },
+  ],
+  "run-a-node": [
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 2, correctAnswer: 2 },
+  ],
+  stablecoins: [
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 3 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 4 },
+  ],
+  defi: [
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 2 },
+    { totalAnswers: 4, correctAnswer: 1 },
+    { totalAnswers: 4, correctAnswer: 4 },
+    { totalAnswers: 4, correctAnswer: 1 },
+  ],
+}
+
+const charFromIdx = (idx: number) => String.fromCharCode(97 + idx)
+
+// Reduce to satisfy QuestionBank data shape for use in front end
+const questionBank = Object.entries(questionBankConfig).reduce(
```

Comment:
@corwintines @wackerow we've introduced a bug here by not allowing us to reference different explanations [like here](https://github.com/ethereum/ethereum-org-website/pull/14208/files#r1881113637)

Reply:
I have some thoughts on a way to easily override and re-use explanations, will patch

Reply:
@wackerow could be also a idea to add a test for this case in the quiz stories.